IoTEdgeModbus/Modules/AzureIoTEdgeModbus/Slave/SessionsHandle.cs

```diff
@@ -48,18 +48,18 @@ public async Task<SessionsHandle> CreateHandleFromConfiguration(ModuleConfig con
             return sessionsHandle;
         }
 
-        public List<ModbusSlaveSession> ModbusSessionList = new List<ModbusSlaveSession>();
+        public readonly List<ModbusSlaveSession> ModbusSessionList = new List<ModbusSlaveSession>();
 
         public void Release()
```

Comment:
This method should also be made async. Then await all session.ReleaseSessionAsync calls.